PokeAlarm/Events/MonEvent.py

```diff
@@ -226,6 +238,23 @@ def generate_dts(self, locale, timezone, units):
             'weight': self.weight,
             'size': locale.get_size_name(self.size_id),
 
+            # Attack rating
+            'rating_attack': (
+                self.rating_attack.upper() if self.rating_attack else '-'),
```

Comment:
```python
'rating_attack': Unknown.is_empty(self.rating_attack, Unknown.TINY),
```
If we are always going to call .upper, we should just send the upper the version? Also we need to consider other locales (although perhaps not an issue, we can get some feedback from @MARTINI1 on this)

Reply:
I'm already sending it to upper. I put it for some forks whose didn't do that maybe.
btw, there is no need to add locales cause attack rating is something like A+ // A- // B+ etc